lmp/bb-build.sh

```diff
@@ -15,8 +15,11 @@ bitbake -p
 bitbake -e > ${archive}/bitbake_global_env.txt
 bitbake -e ${IMAGE} > ${archive}/bitbake_image_env.txt
 
-# Setscene (cache), failures not critical
-bitbake --setscene-only ${IMAGE} || true
+# Before kiskstone (lmp v87) this is need to avoid build failures
```

Comment:
Typo: s/kiskstone/kirkstone/g, this is needed

Reply:
I corrected it and emphasized the lmp version, which is what interests us